pkg/providers/instance/types.go

```diff
@@ -53,11 +54,14 @@ func NewInstance(out *ec2.Instance) *Instance {
 		}),
 		SubnetID: aws.StringValue(out.SubnetId),
 		Tags:     lo.SliceToMap(out.Tags, func(t *ec2.Tag) (string, string) { return aws.StringValue(t.Key), aws.StringValue(t.Value) }),
+		EFAEnabled: lo.ContainsBy(out.NetworkInterfaces, func(ni *ec2.InstanceNetworkInterface) bool {
+			return ni != nil && lo.FromPtr(ni.InterfaceType) == "efa"
```

Comment:
Can we check if `efa` is a constant value that's represented somewhere by the `ec2` package. If not, can we consider creating our own constant?

Reply:
I don't believe it is but I'll verify. Where do you think it would be best for this constant to live if we define it ourself?

Reply:
Maybe next to the resource type in the `apis` directory? Seems reasonable to me for these to live alongside each other for now until there's a more natural home for it long-term.